docs/sources/reference/components/prometheus/prometheus.exporter.windows.md

```diff
@@ -245,8 +245,11 @@ By default, `text_file_directory` is set to the `textfile_inputs` directory in t
 For example, if {{< param "PRODUCT_NAME" >}} is installed in `C:\Program Files\GrafanaLabs\Alloy\`,
 the default will be `C:\Program Files\GrafanaLabs\Alloy\textfile_inputs`.
 
-When `text_file_directory` is set, only files with the extension `.prom` inside the specified directory are read. Each `.prom` file found must end with an empty line feed to work properly.
+When `text_file_directory` is set, only files with the extension `.prom` inside the specified directory are read. 
 
+{{< admonition type="note" >}}
+Each `.prom` file found must end with an empty line feed to work properly.
```

Comment:
Should this be more explicit? Instead of 'to work properly', 'or it will not be read'

Reply:
I can update it to be more explicit.  The change was purely to convert to a note.. .Good timing to improve the content :-)